### PR TITLE
Improve exception handling

### PR DESCRIPTION
**Pull request by VinceMacBuche**

### CODE REVIEW

File: cli/rudder-cli [hunk 1]
```diff
@@ -378,7 +378,7 @@ if __name__ == "__main__":
         print(endpoint.debug_query)
       print("Error in calling " + function.__name__)
       pprint(e)
-      pprint(e.response.text)
+      pprint(e.rudder_message)
```
Comment (peckpeck) [inline cli/rudder-cli]: Can you rewrite RudderError instead to use the standard message field in Exceptions ?

**Comment by peckpeck** [comment]
You should open a redmine ticket for patches now.

**Comment by peckpeck** [comment]
@VinceMacBuche is there a matching ticket ?

**Comment by FlorianHeigl** [comment]
ping, idk if there is a ticket but this PR is stuck over a year.

**Comment by VinceMacBuche** [comment]
Replaced by #12

**Pull request closed without merging**